### PR TITLE
(DOCS-3234) Create AWS manual setup guide

### PR DESCRIPTION
**Pull request by buraizu**
### What does this PR do?
Adds a guide for manually setting up the AWS integration.

### Motivation
DOCS-3234

### Preview
https://docs-staging.datadoghq.com/bryce/create-aws-manual-setup-guide/integrations/guide/aws-manual-setup/

### Additional Notes
Once merged, will update the main AWS docs to point to this guide.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/integrations/guide/aws-manual-setup.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+### Generate an external ID
+Generate an External ID in the <a href="https://app.datadoghq.com/account/settings#integrations/amazon_web_services" target="_blank">AWS integration tile</a> on the integrations page. This is used in the trust policy of the AWS IAM role you create for Datadog.
+1. Select the `Configuration` tab, then `Role Delegation`.
+2. Click `Manually`. This creates an AWS External ID which is used for configuration of the AWS IAM role. For more information about the External ID, see the [IAM User Guide][2].
+3. Copy this value to your clipboard or notepad.  
+  **Note: Do not close the integration tile or the Datadog application page**, as this causes the external ID value to reset.
+
+### AWS IAM Policy for Datadog
+Create an IAM policy for the Datadog role in your AWS account with [all permissions](#datadog-permissions) to take advantage of every AWS integration offered by Datadog. As other components are added to an integration, these permissions may change.
```
Comment (warrierr) [inline content/en/integrations/guide/aws-manual-setup.md:54]:
```suggestion
Create an IAM policy for the Datadog role in your AWS account with the [necessary permissions](#datadog-permissions) to take advantage of every AWS integration offered by Datadog. As other components are added to an integration, these permissions may change. 
```

Comment (warrierr) [reply content/en/integrations/guide/aws-manual-setup.md:54]: @buraizu also this should link to the the AWS Integration IAM Policy, not just the permissions section
https://docs-staging.datadoghq.com/bryce/create-aws-manual-setup-guide/integrations/guide/aws-manual-setup/#aws-integration-iam-policy

File: content/en/integrations/guide/aws-manual-setup.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  text: "Announcing Datadog Security Monitoring"
+---
+
+## Overview
+
+Use this guide to manually set up the Datadog [AWS Integration][1].
+
+Setting up the AWS integration manually involves creating an IAM policy and IAM role in your AWS account, and configuring the role with an AWS External ID generated in your Datadog account. This allows Datadog's AWS account to query AWS APIs on your behalf, and pull data into your Datadog account. The sections below detail the steps for creating each of these components, and then completing the setup in your Datadog account.
+
+## Setup
```
Comment (warrierr) [inline content/en/integrations/guide/aws-manual-setup.md:44]: @buraizu one major structural change, is that I think we should have the `Access keys: Used for GovCloud or China only` manual setup instructions in the same page. I think it makes sense to have  "Role delegation" and 
"Access keys (GovCloud or China Only)" as tab options with different instructions. Both of these are manual setups. I don't think it makes sense to have another guide for manual setup for govcloud and china.

Comment (buraizu) [reply content/en/integrations/guide/aws-manual-setup.md:47]: Thanks for pointing this out! Updated in ac38017

File: content/en/integrations/guide/aws-manual-setup.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+4. Create a new policy in the AWS [IAM Console][3].
+5. Select the `JSON` tab. Paste the [permission policies](#aws-integration-iam-policy) in the textbox. 
+6. Click `Next: Tags` and `Review policy`.
+7. Name the policy `DatadogAWSIntegrationPolicy` or one of your own choosing, and provide an apt description.
+8. Click `Create policy`.
+
+### AWS IAM Role for Datadog
+Create an IAM role for Datadog to use the permissions defined in the IAM policy.
+
+10. Create a new role in the AWS [IAM Console][5].
```
Comment (warrierr) [inline content/en/integrations/guide/aws-manual-setup.md:68]: @buraizu step 9 gets skipped.  Goes from 8 -> 10

File: content/en/integrations/guide/aws-manual-setup.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
+---
+title: AWS Manual Setup Guide
+kind: guide
+description: "Steps for manually setting up the Datadog AWS Integration"
+further_reading:
+- link: "https://docs.datadoghq.com/integrations/amazon_web_services/"
+  tag: "Documentation"
```
Comment (warrierr) [inline content/en/integrations/guide/aws-manual-setup.md:7]: @buraizu could we link to the AWS troubleshooting guide here?

File: content/en/integrations/guide/aws-manual-setup.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+## Overview
+
+Use this guide to manually set up the Datadog [AWS Integration][1].
+
+{{< tabs >}}
+{{% tab "Role delegation" %}}
+
+Setting up the AWS integration manually involves creating an IAM policy and IAM role in your AWS account, and configuring the role with an AWS External ID generated in your Datadog account. This allows Datadog's AWS account to query AWS APIs on your behalf, and pull data into your Datadog account. The sections below detail the steps for creating each of these components, and then completing the setup in your Datadog account.
+
+## Setup
```
Comment (warrierr) [inline content/en/integrations/guide/aws-manual-setup.md:47]: @buraizu can you add the "Setup" header to the top of the govcloud section for consistency between the 2 tabs?

File: content/en/integrations/guide/aws-manual-setup.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+#### AWS
+
+1. In your AWS console, set up an IAM user to be used by the Datadog integration.
+2. Generate an access key and secret key for the Datadog integration IAM user.
+
+For more details, see the [How to use an external ID when granting access to your AWS resources to a third party][1] AWS documentation.
+
+#### Datadog
+
+1. Open the [AWS integration tile][2].
```
Comment (warrierr) [inline content/en/integrations/guide/aws-manual-setup.md:100]: @buraizu can this one start at #3?, like we do for roll delecation?

File: content/en/integrations/guide/aws-manual-setup.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+**Note:** The role name you enter in the integration tile is case sensitive and must exactly match the role name created on the AWS side.
+22. If there is a [Datadog is not authorized to perform sts:AssumeRole][6] error, make sure your AWS trust policy's `sts:ExternalId:` matches the `AWS External ID` previously created in the integration tile.
+23. Click **Install Integration**.
+24. Wait up to 10 minutes for data to start being collected, and then view the out-of-the-box [AWS overview dashboard][7] to see metrics sent by your AWS services and infrastructure.
+{{% /tab %}}
+
+{{% tab "Access keys (GovCloud or China Only)" %}}
+#### AWS
+
+1. In your AWS console, set up an IAM user to be used by the Datadog integration.
```
Comment (warrierr) [inline content/en/integrations/guide/aws-manual-setup.md:93]:
```suggestion
1. In your AWS console, create an IAM user to be used by the Datadog integration with the [necessary permissions](#aws-integration-iam-policy).
```

**Review by warrierr — CHANGES_REQUESTED** [review]
@buraizu my final comments. This LGTM after those are addressed.

File: content/en/integrations/guide/aws-manual-setup.md (hunk 3 above)
Comment (buraizu) [reply content/en/integrations/guide/aws-manual-setup.md]: Thanks for pointing this out! Corrected the numbering across all three sections in #66bf0b7.

**Review by warrierr — APPROVED** [review]
LGTM

File: content/en/integrations/guide/aws-manual-setup.md [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+---
+
+## Overview
+
+Use this guide to manually set up the Datadog [AWS Integration][1].
+
+{{< tabs >}}
+{{% tab "Role delegation" %}}
+
+Setting up the AWS integration manually involves creating an IAM policy and IAM role in your AWS account, and configuring the role with an AWS External ID generated in your Datadog account. This allows Datadog's AWS account to query AWS APIs on your behalf, and pull data into your Datadog account. The sections below detail the steps for creating each of these components, and then completing the setup in your Datadog account.
```
Comment (apigirl) [inline content/en/integrations/guide/aws-manual-setup.md:48]:
```suggestion
Setting up the AWS integration manually involves creating an IAM policy and IAM role in your AWS account and configuring the role with an AWS External ID generated in your Datadog account. This allows Datadog's AWS account to query AWS APIs on your behalf and pull data into your Datadog account. The sections below detail the steps for creating each of these components and then completing the setup in your Datadog account.
```

File: content/en/integrations/guide/aws-manual-setup.md [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+
+## Setup
+
+### Generate an external ID
+Generate an External ID in the <a href="https://app.datadoghq.com/account/settings#integrations/amazon_web_services" target="_blank">AWS integration tile</a> on the integrations page. This is used in the trust policy of the AWS IAM role you create for Datadog.
+1. Select the `Configuration` tab, then `Role Delegation`.
+2. Click `Manually`. This creates an AWS External ID which is used for configuration of the AWS IAM role. For more information about the External ID, see the [IAM User Guide][2].
```
Comment (apigirl) [inline content/en/integrations/guide/aws-manual-setup.md:55]:
```suggestion
2. Click `Manually`. This creates an AWS External ID, which is used for configuration of the AWS IAM role. For more information about the External ID, see the [IAM User Guide][2].
```

File: content/en/integrations/guide/aws-manual-setup.md [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+13. Click `Next`.
+14. If you've already created the policy, search for it on this page and select it. Otherwise, click `Create Policy`, which opens in a new window, and follow the instructions from [AWS IAM Policy for Datadog](#aws-integration-iam-policy). 
+15. Optionally, attach the <a href="https://console.aws.amazon.com/iam/home#policies/arn:aws:iam::aws:policy/SecurityAudit" target="_blank">AWS SecurityAudit Policy</a> to the role if you would like to use Datadog's [Cloud Security Posture Management product][4].
+16. Click `Next: Tags` and `Next: Review`.
+17. Give the role a name such as `DatadogIntegrationRole`, as well as an apt description. 
+18. Click `Create Role`.
+
+### Complete the setup in Datadog
+
+19. Returning to the AWS integration tile page in your Datadog account that you had open in another tab, enter your AWS Account ID **without dashes**, for example: `123456789012`. Your Account ID can be found in the ARN of the role created for Datadog.
```
Comment (apigirl) [inline content/en/integrations/guide/aws-manual-setup.md:85]:
```suggestion
19. Return to the AWS integration tile page in your Datadog account that you had open in another tab, enter your AWS Account ID **without dashes**, for example: `123456789012`. Your Account ID can be found in the ARN of the role created for Datadog.
```

**Review by apigirl — APPROVED** [review]
just a few small things!

**Pull request merged**